.github/workflows/python-cqa.yml

```
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.10]
```

This is what I meant
```suggestion
        python-version: ["3.7", "3.8", "3.9", "3.10"]
```